.version

```diff
@@ -1 +1 @@
-0.58.0
+0.58.1
```

Comment:
You would have no way to know this, but the CLI moves in minor releases on master and only breaks out patch releases on its release branches (which I only cut when there is a need to freeze feature content for a CSM release), so this should be 0.59.0 (same in `version` below).

Reply:
@erl-hpe I have fixed the version and .version.